src/static/css/page.css

```
@@ -858,6 +861,87 @@ pre {
   width: 100%;
 }
 
+/* Webmention code */
+.reaction-tabs [role="tab"][aria-selected="true"] {
+  z-index: 3;
+  font-weight: bold;
+  border-bottom: 0.5rem solid #a8caba;
+}
+
+.reaction-tabs [role="tab"] {
+  position: relative;
+  z-index: 1;
+  background: white;
+  border: 0;
+  padding: 0rem 0.75rem 0.25rem;
```

Solve tap target issue by making sure it's at least 44px high

```suggestion
  padding: 0.6rem 0.75rem 0.6rem;
```